operations/accounting-profiling.rst

```diff
@@ -42,9 +42,16 @@ The Juju commands to accomplish these steps are:
    $ juju config slurmctld acct-gather-frequency="task=30"
 
 In this scenario, ``slurmctld`` will setup everything needed to collect and
-save the metrics, which include creating a ``slurm`` database in InfluxDB.
+save the metrics, which include creating a database in InfluxDB. The name of
```

Comment:
I think it would flow better to break this up into another sentence:

```suggestion
save the metrics. This includes creating a database in InfluxDB. The name of
```